docs/command-reference.md

```diff
@@ -759,6 +759,7 @@ Flags:
 - :whale: `-m, --message`: Commit message
 - :whale: `-c, --change`: Apply Dockerfile instruction to the created image (supported directives: [CMD, ENTRYPOINT])
 - :whale: `-p, --pause`: Pause container during commit (default: true)
+- :whale: `--compression`: Commit compression algorithm,(supported value: zstd or gzip) (default: gzip)
```

Comment:
Does Docker have this? If not this has to be `:nerd_face:`

Reply:
https://docs.docker.com/reference/cli/docker/container/commit/

Docker current is not support. @thaJeztah Not sure if I can consult you? Will docker commit add this feature?

Reply:
HI @lengrongfu , 
If it's `nerd_face`, it can be changed to 
```
- :nerd_face: `--compression`: Commit compression algorithm,(supported value: zstd or gzip) (default: gzip)
```

Reply:
update done.